spec/factories/events.rb

```diff
@@ -7,6 +7,11 @@
     summary     'summary'
     description 'description'
 
+    factory :event_today do
+      name 'event today'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.